etc/config/c.amazon.properties

```diff
@@ -837,11 +837,11 @@ compiler.carm64gtrunk.name=ARM64 gcc trunk
 compiler.carm64gtrunk.semver=trunk
 
 # GCC Morello
-group.cgcc64arm.groupName=Arm Morello GCC
-group.cgcc64arm.compilers=carm64gm101a1
-group.cgcc64arm.isSemVer=true
-group.cgcc64arm.objdumper=/opt/compiler-explorer/arm64morello/gcc-aarch64-none-elf-10.1.morello-alp1-x86_64/aarch64-none-elf/bin/objdump
-group.cgcc64arm.instructionSet=aarch64
+group.cgcc64armmorello.groupName=Arm Morello GCC
```

Comment:
The fix is --- there already was a `group.cgcc64arm` which had the arm 64 compilers. This is a separate group; so I am renaming it and adding the group to the list of ARM groups